core/src/main/java/org/apache/iceberg/deletes/Deletes.java

```diff
@@ -192,27 +185,43 @@ public static PositionDeleteIndex toPositionIndex(CloseableIterable<Long> posDel
     }
   }
 
+  /**
+   * @deprecated since 1.7.0, will be removed in future.
```

Comment:
```suggestion
   * @deprecated since 1.7.0, will be removed in 1.8.0.
```

Reply:
please also update the others

Reply:
@wypoon can we close https://github.com/apache/iceberg/pull/11177?

Reply:
Updated the deprecation comments. Closed https://github.com/apache/iceberg/pull/11177.
Thanks @nastra for reviewing.